book/website/collaborating_github/reviewing-contributions.md

```diff
@@ -0,0 +1,89 @@
+
+- Some start the project and are generally, the owners of the project. The Codeowners who have the rights over who can make reviews and suggest changes to contributions. The individuals or teams who are codeowners for a certain project are specified at the time of project creation in a [file](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) of the project.  
+They can make other contributors as maintainers, reviewers or members working on the project. They can also involve other codeowners and are responsible for granting permissions. 
+
+- Maintainers are involved with creating issues and pull requests whenever required. They keep the codebase updated and help in reviewing contributions. They can merge pull requests.  
+They can also request reviews from someone else. There are various [levels of permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization) for a project in GitHub.
+
+## Reviewing process
+- Any code base or set of changes must be reviewed before merging. If the project os co-owned by many people, the review process is done by handling it in the form of a [code review assignment](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-code-review-assignment-for-your-team) in which certain members of the team are chosen for the review.  
+This member subset of team is selected with the help of algorithms after reviewing is enabled and review assignment is configured for a particular project.  
+- The review process is dependent on the testing of a particular pull request by a maintainer or admin. The admins can easily [test](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally) the changes made in their local systems and suggest changes required.  
+- After this any set of changes through pull requests can be reviewed. When the review process is completed the reviews are [approved](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/approving-a-pull-request-with-required-reviews) or [dismissed](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/dismissing-a-pull-request-review) according to the outcome. 
+
+## Merging changes
+Pull requests created by contributors can be easily merged by maintainers after the review. There are different ways of merging a PR:
+- Merge pull request- Merges all commits to base branch. This keeps all commits made separated. 
+- Squash and merge- Squashes all commits into one and then merges the pull request. 
+- Rebase and merge- Rebases all commits individually to base branch before merging.  
+In the prompt message, the maintainer can provide a merge message along with comments(if any) and then press the “confirm merge” button.
+
+## Guidelines for Review Process and Maintainance
```

Comment:
I think this section would benefit from mentioning the Code of Conduct (and maybe link to what that is, either within this book or elsewhere) and about how it describes the kind of communication and discourse that is appropriate within the project. This should be a contributors first port of call.

Reply:
The guidelines are for maintainers but since I have mentioned about communication and collaboration between the maintainers and the contributors, I better mention it here. I have modified the brief paragraph at the start of this section in [this commit](https://github.com/alan-turing-institute/the-turing-way/pull/1189/commits/c3a05c09ac39fd7a75a8cb339e81e3e6c28c6c3a).  :+1:

Reply:
Looks fabulous, thank you!